MCprep_addon/MCprep_resources/mcprep_data_update.json

```diff
@@ -1002,7 +1004,7 @@
 			"big_smoke_7": "particle/big_smoke_7",
 			"big_smoke_8": "particle/big_smoke_8",
 			"big_smoke_9": "particle/big_smoke_9",
-			"birch": "entity/chest_boat/birch",
+			"birch": "entity/signs/birch",
```

Comment:
Same thing here. Perhaps some paths were changed, but I'd rather we check and verify that this is valid